website/public/js/controllers/groupsCtrl.js

```diff
@@ -47,6 +47,11 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
       group._editing = false;
     };
 
+    $scope.cancelSave = function(group) {
+      group = group.$get();
```

Comment:
You shouldn't need to reassign the variable, just doing:

```js
$scope.cancelSave = function(group) {
  group.$get();
  //...
```

Should work.


That said, I think making a local copy is probably preferable so we don't need to make an extra server call for the cancel activity to work. This will probably mean making an `$scope.editGroup` function.